### PR TITLE
closes #347 Dark theme is not uniform 

### LINKED ISSUE #347: Dark theme is not uniform
**Actual Behaviour**

Dark theme is not getting applied to all the pages like view image, edit image, share, select images.......

**Expected Behaviour**

Theme should be uniform

**Would you like to work on the issue?**

yes

### PR DESCRIPTION
**Pull request by vinaysajjanapu**
Fixes issue #347 

Changes: 

Added settheme functions to all activities and changes layout attributes(when necessary).
Corrected a wrong value in color resources

### CODE REVIEW

**Comment by phimpme-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">:tada: Thanks for the PR @vinaysajjanapu we hope you have changed the variables in the upload_apk_forks.sh its for easy moderation.</td>
    </tr>
  </tbody>
</table>

:link: Maintainers: the apk to test can be found at https://github.com/vinaysajjanapu/phimpme-android/blob/apk/app-debug.apk

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by Abhi2424shek** [comment]
The diff looks fine, but please do check the failing checks.

**Comment by vinaysajjanapu** [comment]
I checked. Everything seemed fine. Can't understand the reason for error. Help needed.

File: app/src/main/java/vn/mbm/phimp/me/ImagePickerActivity.java [hunk 1]
```diff
@@ -4,18 +4,21 @@
 import android.content.DialogInterface;
 import android.content.Intent;
 import android.content.res.Configuration;
+import android.content.res.Resources;
```
Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/ImagePickerActivity.java]: Unused input

File: app/src/main/java/vn/mbm/phimp/me/ImagePickerActivity.java [hunk 2]
```diff
@@ -4,18 +4,21 @@
...
 import android.database.Cursor;
 import android.net.Uri;
 import android.os.AsyncTask;
 import android.os.Bundle;
 import android.provider.MediaStore;
+import android.support.annotation.ColorInt;
```
Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/ImagePickerActivity.java]: Unused input

File: app/src/main/java/vn/mbm/phimp/me/ImagePickerActivity.java [hunk 3]
```diff
@@ -4,18 +4,21 @@
...
 import android.support.annotation.NonNull;
 import android.support.v7.app.AlertDialog;
 import android.support.v7.widget.GridLayoutManager;
 import android.support.v7.widget.RecyclerView;
 import android.support.v7.widget.StaggeredGridLayoutManager;
 import android.support.v7.widget.Toolbar;
 import android.util.DisplayMetrics;
+import android.util.TypedValue;
```
Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/ImagePickerActivity.java]: Unused input

**Pull request merged**